### ISSUE TITLE
Catcher does not speed up if hold shift when beatmap is loading

### ISSUE THREAD

**Issue by plankp** [ppy/osu#3122]
In osu catch when the beatmap is loading, hold down shift, and do not release it.

After the beatmap has loaded, (shift is still held down,) pressing left and right buttons will not cause the catcher to speed up (sprint).

Or is this behaviour intended?

**Comment by ekrctb** [comment]
This behavior is described in ppy/osu-framework#1682. I think it should be fixed.

**Comment by plankp** [comment]
Still broken for me in the 2018.730.0.

**Comment by ekrctb** [comment]
Yes, the PR I mentioned above didn't fix this issue. I just noted the behavior.
